pkg/apiserver/registry/networkpolicy/clustergroupmember/rest_test.go

```diff
@@ -162,3 +214,133 @@ func TestRESTGet(t *testing.T) {
 		assert.Equal(t, tt.expectedObj, actualGroupList)
 	}
 }
+
+func TestRESTGetPagination(t *testing.T) {
+	tests := []struct {
```

Comment:
I've modified the test cases, (total 3 members) one of `page 5, limit 2` where page index exceeds actual, one of `page 1, limit -2` where limit is invalid.
If I understand your comment correctly, in both cases, the current design is to return full list instead of throwing an error. Let me know if returning an error works better here, but I checked [K8s List pagination](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L591) it only pages for `limit>0`, otherwise it's treated as normal.
For invalid parameters like "providing string for int"/"key not found", these are handled by the conversion function generated by `+k8s:conversion-gen:explicit-from=net/url.Values`, not tested in UT.

Reply:
So there is no error either if page < 0?